packages/internal/test/test-utils.js

```diff
@@ -68,13 +68,21 @@ test('makeMeasureSeconds', async t => {
 });
 
 test('assertAllDefined', t => {
-  /** @type {{s: string, m?: string | undefined}} */
+  /** @type {{ s: string, m: string | undefined, u?: string}} */
   const obj = { s: 'defined', m: 'maybe' };
   assertAllDefined(obj);
   // typecheck
   obj.m.length;
+  // @ts-expect-error key presence not checked
+  obj.u.length;
```

Comment:
I think this is actually executed, which fails the test, right?

Reply:
yep